.eslintrc.js

```diff
@@ -1,64 +1,36 @@
+const path = require('path')
+
+const ERROR = 'error'
+const WARN = 'warn'
+const OFF = 'off'
+
 module.exports = {
-  "env": {
-    "browser": true,
-    "es6": true,
-    "jest": true
+  env: {
+    browser: true,
+    jest: true,
   },
-  "globals": {
-    "__DEV__": true
+  extends: ['@instacart/eslint-config'],
+  rules: {
+    'no-plusplus': OFF,
+    // Fails on non literal roles attrs. Set to error after this is released:
+    // https://github.com/evcohen/eslint-plugin-jsx-a11y/pull/572
+    'jsx-a11y/no-static-element-interactions': WARN,
+    'react/no-string-refs': OFF,
+    'react/no-find-dom-node': OFF,
```

Comment:
For these two rules, should we set to Warn instead? That might prevent future usage. We should be using React.createRef to handle these cases.

Reply:
We could -- but I made an assumption that `React.createRef` wasn't used to maintain backwards compatibility? (like legacy portal support for eg)

Reply:
Good point. We plan on implementing a strategy to allow exposing refs for certain components.  When we do that we will be dropping support for legacy portal as well as upgrading our React peer dependency. We could leave it OFF for now, and when we do that process we could revisit the rule overrides.